perf_test/sparse/KokkosSparse_spmv_struct_tuning.cpp

```diff
@@ -438,26 +480,21 @@ int main(int argc, char **argv)
     }
 
     if(compare_cusparse) {
-#if defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE) && defined(KOKKOS_ENABLE_CUDA_LAMBDA)
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE)
       // The data needs to be reformatted for cusparse before launching the kernel.
       // Step one, extract raw data
       using graph_type = typename matrix_type::StaticCrsGraphType;
       using cusparse_int_type = typename Kokkos::View<int*,
 						      typename graph_type::entries_type::array_layout,
-						      typename matrix_type::device_type>;
+						      typename graph_type::device_type>;
 
       typename graph_type::row_map_type::const_type     Arowptr = A.graph.row_map;
       typename graph_type::entries_type::const_type     Acolind = A.graph.entries;
       typename matrix_type::values_type::non_const_type Avals   = A.values;
-      cusparse_int_type Arowptr_cusparse, Acolind_cusparse;
-      Arowptr_cusparse = cusparse_int_type("Arowptr", Arowptr.extent(0));
-      Acolind_cusparse = cusparse_int_type("Acolind", Acolind.extent(0));
-      Kokkos::parallel_for(Arowptr.extent(0), KOKKOS_LAMBDA(const size_t idx) {
-      	  Arowptr_cusparse[idx] = Arowptr[idx];
-      	});
-      Kokkos::parallel_for(Acolind.extent(0), KOKKOS_LAMBDA(const size_t idx) {
-      	  Acolind_cusparse[idx] = Acolind[idx];
-      	});
+      cusparse_int_type Arowptr_cusparse("Arowptr", Arowptr.extent(0));
+      cusparse_int_type Acolind_cusparse("Acolind", Acolind.extent(0));
```

Comment:
You shouldn't need to allocate column indices unless they don't have type `int`.

Reply:
That's correct but since this is a test I did not want to write too much logic around the type of the ordinal.
It would make the test a bit faster but this part is not timed and we already know that the cusparse interface is changing in future versions. Ideally this test should be rewritten down the road for the new interface that will make this a bit cleaner...